server/src/test/resources/file.conf

```diff
@@ -0,0 +1,5 @@
+#reduce delay for test
```

Comment:
Is it necessary?

Reply:
yes ,this file is for test in order to reduce asyn-committing-retry-delay (default is 30 second) and timeout-retry-delay (default is 30 second) , then we can fast check event with `TimeoutRollbacking` status will come from core